master/docs/manual/cfg-properties.rst

```diff
@@ -188,7 +188,7 @@ A common mistake is to omit the trailing "s", leading to a rather obscure error
                 util.Interpolate('REVISION=%(prop:got_revision)s'),
                 'dist']))
 
-This example will result in a ``make`` command with an argument like ``REVISION=12098``.
+This example will result in the command: ``make REVISION=12098 dist``.
```

Comment:
I think this does a bit of oversimplification.

the actual command is ``["make", "REVISION=12098", "dist"]`` , which is equivalent to ``make REVISION=12098 dist``, but with shell quoting taken care of (will not harm if got_revision is ``;rm -rf /``).